CRITICAL - 2022-05-11 00:53:11 --> Invalid file: /front/usuarios/registro.php
#0 C:\xampp\htdocs\ProyectoX\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('/front/usuarios...')
#1 C:\xampp\htdocs\ProyectoX\system\Common.php(1089): CodeIgniter\View\View->render('/front/usuarios...', Array, true)
#2 C:\xampp\htdocs\ProyectoX\app\Controllers\Usuario_controller.php(12): view('/front/usuarios...')
#3 C:\xampp\htdocs\ProyectoX\system\CodeIgniter.php(832): App\Controllers\Usuario_controller->create()
#4 C:\xampp\htdocs\ProyectoX\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario_controller))
#5 C:\xampp\htdocs\ProyectoX\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ProyectoX\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-05-11 00:53:13 --> Invalid file: /front/usuarios/registro.php
#0 C:\xampp\htdocs\ProyectoX\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('/front/usuarios...')
#1 C:\xampp\htdocs\ProyectoX\system\Common.php(1089): CodeIgniter\View\View->render('/front/usuarios...', Array, true)
#2 C:\xampp\htdocs\ProyectoX\app\Controllers\Usuario_controller.php(12): view('/front/usuarios...')
#3 C:\xampp\htdocs\ProyectoX\system\CodeIgniter.php(832): App\Controllers\Usuario_controller->create()
#4 C:\xampp\htdocs\ProyectoX\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario_controller))
#5 C:\xampp\htdocs\ProyectoX\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ProyectoX\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-05-11 00:53:57 --> Invalid file: /front/usuarios/registro.php
#0 C:\xampp\htdocs\ProyectoX\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('/front/usuarios...')
#1 C:\xampp\htdocs\ProyectoX\system\Common.php(1089): CodeIgniter\View\View->render('/front/usuarios...', Array, true)
#2 C:\xampp\htdocs\ProyectoX\app\Controllers\Usuario_controller.php(12): view('/front/usuarios...')
#3 C:\xampp\htdocs\ProyectoX\system\CodeIgniter.php(832): App\Controllers\Usuario_controller->create()
#4 C:\xampp\htdocs\ProyectoX\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario_controller))
#5 C:\xampp\htdocs\ProyectoX\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ProyectoX\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2022-05-11 02:39:13 --> Error connecting to the database: Unknown database 'mulas_vittorio'
CRITICAL - 2022-05-11 02:39:13 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'mulas_vittorio'
#0 C:\xampp\htdocs\ProyectoX\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\ProyectoX\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 C:\xampp\htdocs\ProyectoX\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\ProyectoX\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('vittogoya@hotma...', 'email', Array, NULL)
#4 C:\xampp\htdocs\ProyectoX\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'vittogoya@hotma...', Array, Array)
#5 C:\xampp\htdocs\ProyectoX\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 C:\xampp\htdocs\ProyectoX\app\Controllers\Usuario_controller.php(22): CodeIgniter\Controller->validate(Array)
#7 C:\xampp\htdocs\ProyectoX\system\CodeIgniter.php(832): App\Controllers\Usuario_controller->formValidation()
#8 C:\xampp\htdocs\ProyectoX\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario_controller))
#9 C:\xampp\htdocs\ProyectoX\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ProyectoX\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2022-05-11 02:46:53 --> Function name must be a string
#0 C:\xampp\htdocs\ProyectoX\system\CodeIgniter.php(832): App\Controllers\Usuario_controller->formValidation()
#1 C:\xampp\htdocs\ProyectoX\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario_controller))
#2 C:\xampp\htdocs\ProyectoX\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\ProyectoX\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2022-05-11 03:10:05 --> Undefined variable: data
#0 C:\xampp\htdocs\ProyectoX\app\Controllers\Usuario_controller.php(45): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 45, Array)
#1 C:\xampp\htdocs\ProyectoX\system\CodeIgniter.php(832): App\Controllers\Usuario_controller->formValidation()
#2 C:\xampp\htdocs\ProyectoX\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario_controller))
#3 C:\xampp\htdocs\ProyectoX\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ProyectoX\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
